lib/tapioca/generator.rb

```diff
@@ -149,10 +161,23 @@ def build_dsl(requested_constants)
       end
 
       say("")
-      say("Done", :green)
+      if should_verify
+        begin
+          verify_dsl_rbi(tmp_dir: Pathname.new(outpath))
+        rescue OutOfSyncError => e
```

Comment:
I don't love using exceptions for control flow. Also like other languages I'm sure using exceptions in Ruby are slower than its alternatives.

It's only thrown within `verify_dsl_rbi` so I think it can be replaced with something simple such as return value if we don't want to overcomplicate.